images/build/cross/Dockerfile

```diff
@@ -41,8 +41,11 @@ ENV KUBE_CROSSPLATFORMS \
 ##------------------------------------------------------------
 
 # Pre-compile the standard go library when cross-compiling. This is much easier now when we have go1.5+
-RUN for platform in ${KUBE_CROSSPLATFORMS}; do GOOS=${platform%/*} GOARCH=${platform##*/} go install std; done \
-    && go clean -cache
+RUN targetArch=$(echo $TARGETPLATFORM | cut -f2 -d '/') \
```

Comment:
nitpick: how would you feel about setting `TARGETARCH` as env var once at beginning so that we don't have to execute on each command?

Reply:
Apparently that does not work, i ended up eye-balling a bunch of code similar:
https://grep.app/search?q=TARGETPLATFORM&case=true&filter[lang][0]=Dockerfile

could be something wrong in my setup. could we merge and iterate please?

Reply:
works for me 👍